### PR TITLE
Add correct references for connecting Splunk to TFC.

### PR DESCRIPTION
**Pull request by glenngillen**
Clarifying the slight nuance in configuring the Splunk App for Splunk Cloud customers.

### CODE REVIEW

**Comment by glenngillen** [comment]
@caseylang remember that thing we did a long time ago that I said I'd update the docs for. And then I did. But... I forgot to create an actual PR for you to review.

Here we are. 🤦

File: content/source/docs/cloud/integrations/splunk/index.html.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ The current release of the Terraform Cloud for Splunk app supports the following
 ## Installation & Configuration
 
 * Install the [Terraform Cloud for Splunk app via Splunkbase](https://splunkbase.splunk.com/app/5141/)
+  * Splunk Cloud users should consult the latest instructions on how to [use IDM with cloud-based add-ons](https://docs.splunk.com/Documentation/SplunkCloud/8.0.2007/Admin/IntroGDI#Use_IDM_with_cloud-based_add-ons) within the Splunk documentation.
```
Comment (nfagerlund) [inline content/source/docs/cloud/integrations/splunk/index.html.md:45]:
```suggestion
    * Splunk Cloud users should consult the latest instructions on how to [use IDM with cloud-based add-ons](https://docs.splunk.com/Documentation/SplunkCloud/8.0.2007/Admin/IntroGDI#Use_IDM_with_cloud-based_add-ons) within the Splunk documentation.
```

If that's meant to be a nested list item under the first one, it should have exactly four spaces.

**Review by nfagerlund — APPROVED** [review]
One markup suggestion, otherwise 👍🏼

**Review by caseylang — APPROVED** [review]

**Comment by nfagerlund** [comment]
With Casey's ✅ , I'm just going to apply the markup fix and merge.

**Pull request merged**